src/scripts/quests/questTypes/UseOakItemQuest.ts

```diff
@@ -41,7 +41,18 @@ class UseOakItemQuest extends Quest implements QuestInterface {
     }
 
     get description(): string {
-        return `Gain the benefit from the ${GameConstants.humanifyString(OakItemType[this.item])} ${this.amount.toLocaleString('en-US')} times.`;
+        const desc = [];
+        desc.push(`Equip the ${GameConstants.humanifyString(OakItemType[this.item])} and`);
+        if (OakItemType[this.item].match('Magic_Ball')) {
+            desc.push(`capture ${this.amount.toLocaleString('en-US')} wild Pokémon.`);
+        } else if (OakItemType[this.item].match('Amulet_Coin')) {
+            desc.push(`earn ${this.amount.toLocaleString('en-US')} PokéDollars.`);
+        } else if (OakItemType[this.item].match('Exp_Share')) {
```

Comment:
Sorry, 1 last thing, could you compare the values against the enum:
```js
if (this.item == OakItemType.Blaze_Cassette) {
```

Reply:
For sure!